scripts/README.md

```diff
@@ -0,0 +1,21 @@
+<h2> General-purpose scripts for Reactome's release </h2>
```

Comment:
Tip: Markdown has its own set of formatting instructions - you don't need to use HTML. Not necessary, but I find it is often slightly faster than HTML. Here's what github supports: https://guides.github.com/features/mastering-markdown/ (not all markdown renderers are the same).